test/node_test_setup.js

```diff
@@ -82,7 +82,7 @@ var virtualWindow = new jsdom.JSDOM(
     resources: new CustomResourceLoader(),
```

Comment:
Do we really need this?
Is just a console log...
And then we don't need to reconfigure the env so we won't need to expose setEnvForTests as well

Reply:
we won't need to expose it in fabricJS, nor remove it, we will probably be able to import it directly from the build folder.

Reply:
I mean we don't need this configuration

Reply:
The CustomResourceLoader justs logs to console a line
It is meaningless

Reply:
So we can use the node env from fabric w/o this headache